game/lib/interactive_menu.py

```diff
@@ -0,0 +1,106 @@
+        if pos_x < 1:
+            suf = pygame.display.get_surface()
+            if suf is not None:
+                self.__pos_x = pos_x * suf.get_width()
+            else:
+                self.__pos_x = DEFAULT_POS_X
+        else:
+            self.__pos_x = pos_x
+        if pos_y < 1:
+            suf = pygame.display.get_surface()
+            if suf is not None:
+                self.__pos_y = pos_y * suf.get_height()
+            else:
+                self.__pos_y = DEFAULT_POS_Y
+        else:
+            self.__pos_y = pos_y
+        if width < 1:
+            suf = pygame.display.get_surface()
+            if suf is not None:
+                self.__width = width * suf.get_width()
+            else:
+                self.__width = DEFAULT_WIDTH
+        else:
+            self.__width = width
+        if height < 1:
+            suf = pygame.display.get_surface()
+            if suf is not None:
+                self.__height = height * suf.get_height()
+            else:
+                self.__height = DEFAULT_HEIGHT
+        else:
+            self.__height = height
+        self.__text_size = text_size
+        self.__text_color = text_color
+        self.__marked_line_color = marked_text_color
+        self.__background_color = background_color
+        self.__content = content
+        self.__text_size = text_size
+        self.__marked_line_index = 0   # index to currently marked line
+        self.__font_text = pygame.font.SysFont(FONT_STYLE, self.__text_size)
+        self.__top_margin = 10
+        self.__amount_to_print = int((self.__height-self.__top_margin)/self.__text_size)
+
+    def check(self, scroll_surface, events):
+        for event in events:
+            if event.type == pygame.KEYDOWN and event.key == pygame.K_UP:
+                if self.__marked_line_index != 0:
+                    self.__marked_line_index = self.__marked_line_index - 1
+                    self.fill_surface(scroll_surface)
+                    pygame.display.flip()
+            elif event.type == pygame.KEYDOWN and event.key == pygame.K_DOWN:
+                if self.__marked_line_index != (len(self.__content)-1):
+                    self.__marked_line_index = self.__marked_line_index + 1
+                    self.fill_surface(scroll_surface)
+                    pygame.display.flip()
+
+    def draw(self, events):
+        scroll_surface = pygame.Surface((self.__width, self.__height))
+        scroll_surface.fill(self.__background_color)
+        scroll_rectangle = pygame.Rect(self.__pos_x, self.__pos_y, self.__width, self.__height)
+        display_surface = pygame.display.get_surface()
+        self.check(scroll_surface, events)
+        self.fill_surface(scroll_surface)
+        display_surface.blit(scroll_surface, scroll_rectangle)
+
+    def fill_surface(self, scroll_surface):
+        page = int(self.__marked_line_index/self.__amount_to_print)  # which page of elements should be printed
```

Comment:
Skoro page to jest element który będzie printowany, to zmień nazwę page na printed_page i wywal komentarz